nautobot/dcim/forms.py

```diff
@@ -3893,7 +3893,7 @@ def clean(self):
             raise forms.ValidationError({"length_unit": "Must specify a unit when setting length"})
 
 
-class CableFilterForm(BootstrapMixin, StatusModelFilterFormMixin, forms.Form):
+class CableFilterForm(BootstrapMixin, StatusModelFilterFormMixin):
```

Comment:
`StatusModelFilterFormMixin` already inherits `forms.Form` so I dont think this is necessary.

Reply:
Disagree a bit on this point - because of the way forms.Form works, the mixins do have to inherit from it, but I consider that an implementation detail, rather than something we should assume/rely upon in our usage of these mixins.